app/views/group_offers/index.xlsx.axlsx

```diff
@@ -0,0 +1,73 @@
+wb = xlsx_package.workbook
+
+col_header = wb.styles.add_style(:bg_color => "FFDFDEDF", b: true , alignment: { horizontal: :center, vertical: :center }, border: {
```

Comment:
Use newer hash syntax `bg_color:`

Also IMHO it would look nicer to split it into more lines, i.e.

```ruby
col_header = wb.styles.add_style(
  bg_color: ...
  ...
)
```